pkg/internal/test/test_doubles.go

```diff
@@ -50,6 +52,13 @@ func ExpectPayload(payloadAssert func(payload map[string]interface{}) bool) gock
 	return matcher
 }
 
+//nolint
+func CreateEmptyGitHubClient() *github.Client {
```

Comment:
Maybe worth changing `nolint` to something helpful here? Mentioning what this "empty" really means. Maybe also worth to rethink the naming here for two reasons:

- empty doesnt really say what it does? Is it not calling real GH so it's `noop`/`dummy` implementation?
- how about following the Go convention, it's a constructor function, so `New....Client()` would fit.

I will do the same for the logger below in the separated PR (need some new gh karma :D).

Reply:
https://github.com/arquillian/ike-prow-plugins/pull/79

Reply:
> empty doesnt really say what it does? Is it not calling real GH so it's noop/dummy implementation?

Yes, empty was a really stupid name. 
Technically speaking, it is just a default GH client (without any token set) and it calls real GH, just without any authentication. So I'll call the function `NewDefaultGitHubClient`

Reply:
done 8baf6d3